app/assets/stylesheets/shared/footer.scss

```diff
@@ -23,6 +23,10 @@
     }
   }
 
+  .greyed-out {
+    color: #777;
```

Comment:
Color literals like `#777` should only be used in variable declarations; they should be referred to via variable everywhere else.